wled00/FX.h

```diff
@@ -44,8 +44,20 @@
 
 /* Not used in all effects yet */
 #define WLED_FPS         42
-#define FRAMETIME_FIXED  (1000/WLED_FPS)
 #define FRAMETIME        strip.getFrameTime()
+#if defined(ARDUINO_ARCH_ESP32) && !defined(CONFIG_IDF_TARGET_ESP32C3)    // all ESP32 except -C3(very slow, untested)
+  #define FRAMETIME_FIXED  (strip.getFrameTime() < 10 ? 12 : 24)            // allow faster FRAMETIME_FIXED when target FPS >= 100  
+  #define MIN_SHOW_DELAY   (max(2, (_frametime*5)/8))                       // supports higher framerates and better animation control -- 5/8 = 62%
```

Comment:
I do not know why @Aircoookie introduced `MIN_SHOW_DELAY` but by examining the code IMO it is intended to postpone `strip.service()` if minimum time has not yet passed.
I'm not sure if it is ok to be variable. It may not give other tasks enough time.

Reply:
That's the reason I kept the old MIN_SHOW_DELAY for 8266 and for -C3.

Making the delay variable (depending on target FPS) works on esp32 - its tested in the MoonModules  forks and we have a similar variable delay for more than a year already.

Reply:
I'm somewhat leery of this change, too - there are a couple of FX that use this define, changing it will change their timing.  (Though probably the FX should be updated to call the strip method instead).

I'd also note that you do still have a fixed "minimum show delay" of 2 in the new ESP32 logic, expressly tested for on https://github.com/Aircoookie/WLED/blob/451cd4c74aff2faff9ab11f746f9e454ab10bd2e/wled00/FX_fcn.cpp#L1316 .